schema.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
So, the point is to extend this schema, right ?

Reply:
I think so, yes. With all the remaining fields/types, I suppose.